src/main/java/net/quickwrite/fluent4j/ast/FluentBase.java

```diff
@@ -207,6 +207,10 @@ private StringSlice getVariantIdentifier(final StringSlice content) {
 
     @Override
     public CharSequence getResult(final DirectFluentBundle bundle, final FluentArgs arguments) {
+        if (this.fluentElements.size() == 1) {
+            return this.fluentElements.get(0).getResult(bundle, arguments);
+        }
```

Comment:
When there is this simple check there is the question if there should be another check if nothing is in the list or if it should just be handled by the rest of the code.

Reply:
It is probably not a good idea to add another condition as it doesn't really add a big benefit as when there is nothing in the message it probably won't be accessed that often and when it gets accessed it is just the creation of the `StringBuilder` as it skips the loop completely.

Reply:
And so I don't think it is a good idea to add this check.